src/components/forms/MultiElement.js

```diff
@@ -0,0 +1,239 @@
+import React, { PureComponent } from "react";
+import PropTypes from "prop-types";
+import { Form } from "./Form";
+import _ from "lodash";
+import { copyStateAndSetValuesToNull } from "../../utils/general";
+
+/**
+ * Types
+ * @typedef {Object} Element:
+ *		@property {String}                         placeHolder             Placeholder that will be shown inside of each element.
+ *		@property {String}                         camelCaseName           Unique name in camelCase format.
+ *		@property {("DatePicker"|"FilterList"
+ *		           |"Input"|"PickerInput"
+ *		           |"TimePicker"|"MultiElement")}  type                    Type of element.
+ *		@property {boolean=}                       isRequired              Optional field to determine if element should be required.
+ *		@property {Options=}                       options                 Optional fields for element-specific functionality.
+ *		@property {ConditionalValue=}              conditionalValues       Optional object that contains element:value pairs that depend on the current element.
+ *		@property {Function=}                      isValid                 Optional function that checks if the current element Value is valid.
+ *		@property {String=}                        validShape              Optional string that tells the user how the input should look.
+ * @typedef {Object} ConditionalValue:
+ *		@property {Function}                       name                    (camelCaseName) of the element whose value is going to be modified.
+ *		@property {Function}                       value                   Function is used to obtain the new value for the (camelCaseName) element.
+ * @typedef {Object} Options:
+ *		@property {Function}                       showIfParentValueEquals Function checks the parent value and returns whether the element should be shown.
```

Comment:
Use 2 spaces (instead of 1) to separate type from name and name from description